index.js

```diff
@@ -83,7 +112,15 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  // let movieObj = {}
+  // for (let i = 0; i < movies.length; i ++){
+  //   if (movies[i].imdbID = id){
+  //     return movies[i].title
+    // } 
+    return null
```

Comment:
This is very close!  What is the difference between `=` and `===` in js (line 118).